apps/mull-ui/src/app/components/custom-file-upload/custom-file-upload.tsx

```diff
@@ -30,6 +32,7 @@ export const CustomFileUpload = ({
             <FontAwesomeIcon className="custom-file-upload-icon" icon={faImages} />
           </div>
         )}
+        {isEditProfile && <FontAwesomeIcon className="pencil-icon" icon={faPencilAlt} />}
```

Comment:
💡Maybe a name like `displayPencilIcon` would be better. We might want to have that pencil icon in places other than the profile edit page.

Also, careful with a classname like `pencil-icon`. Might be good to namespace it, something like `custom-file-upload-pencil-icon`